fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -268,6 +268,8 @@ protected void configure() {
 
                 DataSourceMetadataService dataSourceMetadataService = buildDataSourceMetadataService();
 
+                bind(dataSourceMetadataService).to(DataSourceMetadataService.class);
```

Comment:
What were we doing to have this service injected in other places? Was it only being used by us to inject it, and never actually bound for others to use?